website/docs/routes/bbs.mdx

```diff
@@ -254,6 +254,12 @@ If you opt to enable `fulltext` feature, consider adding `limit` parameter to yo
 
 <Route author="TonyRL" example="/pikabu/user/@Eujenu" path="/pikabu/user/:name" paramsDesc={['User name']} radar="1" notOperational="1" />
 
+## PKMer {#pkmer}
+
+### 最近更新 {#pkmer-zui-jin-geng-xin}
+
+<Route author="Gnoyong" example="/pkmer/recent" path="/pkmer/recent" notOperational="1" radar="1" />
```

Comment:
Do not add `notOperational` if it works.

Reply:
> Do not add `notOperational` if it works.

👀